### ISSUE TITLE
[v2] Missing tab icons

### ISSUE THREAD

**Issue by TehTux** [causefx/Organizr#1278]
###### Operating System: Windows/MacOS/Ubuntu
<hr>

##### Problem Description:
Can you add tab icons for:

- FRITZ!Box
- Kodi
- openHAB

Thank You!

**Comment by HalianElf** [comment]
You can upload your own in the image manager in the settings.

**Comment by TehTux** [comment]
> You can upload your own in the image manager in the settings.

I know, and I did, too. But I think it is used by many users, so it would be nice to have it by default.

**Comment by tronyx** [comment]
Will have to leave it up to Cause because I've honestly only heard of one of the items in that list, Kodi.

**Comment by GilbN** [comment]
@TehTux just make a pr.

**Comment by jscottelblein** [comment]
On a related note: has anyone ever optimized the default .png icons for Organizr?

A while back while still on v1 I ran all of the ones that come with it through [pnggauntlet ](https://pnggauntlet.com/) and it optimized many of them. I kept them locally but think it might be nice to have them all optimized by default?

I always make a habit of giving my pngs a run through pnggauntlet, I've never noticed a negative difference.
